config/sipi.config.lua

```diff
@@ -49,6 +49,29 @@ sipi = {
     nthreads = 8,
 
     --
+    -- SIPI is using libjpeg to generate the JPEG images. libjpeg requires a quality value which
+    -- corresponds to the compression rate. 100 is (almost) no compression and best quality, 0
+    -- would be full compression and no quality. Reasonable values are between 30 and 95...
+    --
+    jpeg_quality = 60,
+
+    --
+    -- For scaling images, SIPI offers two methods. The value "high" offers best quality using expensive
```

Comment:
Would you mind creating a branch to which you could apply your suggestions? I propose to create that branch from `develop` once `fragmentarium` is merged.